### ISSUE TITLE
Probably bug and proposition: Social media icons are not shown

### ISSUE THREAD

**Issue by ivanlucky22** [AnthonyNahas/ngx-auth-firebaseui#84]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ x] feature request
```

### Versions
ngx-auth-firebaseui v1.1.0

I've tried to use component ngx-auth-firebaseui-providers with [theme]="themes.MINI_FAB" and faced the following issue:
![image](https://user-images.githubusercontent.com/8539337/41370581-ba0357f0-6f50-11e8-9ba6-c70e0d9744a0.png)

mat-icons do not show social network logos. I have the following structure in the assets:
![image](https://user-images.githubusercontent.com/8539337/41370691-f76db0ae-6f50-11e8-8734-fecf1567dce9.png)

Could you please take a look, am I doing something wrong?

As an alternative I would recommend using font-awesome set of [icons](https://fontawesome.com/v4.7.0/icons/). It seems that it has all popular social media icons and sometimes it's even easier to integrate to your project than dealing with SVG icons.

**Comment by AnthonyNahas** [comment]
did you import the assets into your `angular.json` ?

--> 

```json
"assets": [
              "src/assets",
              "src/favicon.ico",
              {
                "glob": "**/*",
                "input": "node_modules/ngx-auth-firebaseui/assets",
                "output": "/assets"
              }
            ],
```

see that in [readme](https://github.com/AnthonyNahas/ngx-auth-firebaseui#4-additional-requirements---assets) 

tell me if that works! take a look to the demo app inside the repo [here](https://github.com/AnthonyNahas/ngx-auth-firebaseui/blob/master/demo/angular.json#L23-L25) 

related to the question about font-awesome... 

`ngx-auth-firebaseui` is built completely with material deisgn, so I need material design icons too.

In future work, I will maybe support `bootstrap` as an extra module!
Then I will make use of the font-awesome.

Note: I am trying to find a solution so that the icons are directly imported in the library without to reply on an import via angular.json...

Tell me if that works!

cheers

**Comment by ivanlucky22** [comment]
@AnthonyNahas , it didn't work, but I found more details. The case is complex, so some explanations needed:
I have 2 routes (pages): main page and profile page. In profile page I used <ngx-auth-firebaseui> with all of the providers and all social media icons are present. On the main page I'm using only <ngx-auth-firebaseui-providers> and there are no icons.
But If I will navigate to profile page and then back to main page - the icons for social media appear. It seems that code for registering of icons is present only in <ngx-auth-firebaseui> component and only after <ngx-auth-firebaseui> was rendered icons become available for <ngx-auth-firebaseui-providers> component too. Do you think it makes sense?

**Comment by AnthonyNahas** [comment]
Yes this makes sense! 
I found the bug and I will fix that in the next patch `v1.1.0`!

The problem is that the icons are registered within the authentication component! `<ngx-auth-firebaseui>` [take a further look](https://github.com/AnthonyNahas/ngx-auth-firebaseui/blob/master/src/module/components/auth/auth.component.ts#L68-L80)

So if you are lazy loading a module, and this component is not being injected, the target material icons will never be registered! 


I am 99% sure! Thanks for reporting!

**Comment by AnthonyNahas** [comment]
![bug 84](https://user-images.githubusercontent.com/20149774/41485486-fc4be8a2-70e0-11e8-8328-c3aa3575218b.gif)

I was able to reproduce the bug!

I am writing now unit test for the `ngx-auth-firebaseui-providers` compoenent!
The fix is on the way

👍

**Comment by ivanlucky22** [comment]
Thanks!
